server/apps/distributed-app/sample-configuration/queue.properties

```diff
@@ -0,0 +1 @@
+queue.choice=PULSAR
```

Comment:
Should be RabbitMQ by default IMO.
Maybe we should create another `queue.properties` with Pulsar choice and volume it into the pulsar docker compose?

Reply:
BTW did we document this `queue.properties` somewhere?

Reply:
No, we haven't had any document for this

Reply:
Can you try to add that documentation?

Reply:
Ok, I"ll do it